### PR TITLE
Add modules from published build info to command summary

### PR DESCRIPTION
**Pull request by RobiNino**
- [x] All [tests](https://github.com/jfrog/jfrog-cli-core#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-cli-core/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----
Depends on https://github.com/jfrog/build-info-go/pull/262
Based on https://github.com/jfrog/jfrog-cli-core/pull/1208

If a build info is published, its modules will show as artifact trees in GitHub job summary.
If no build info is published, only the generic artifact tree will be shown.

Fixed link to artifacts for projects and Artifactory 6.

Preview:
image

### CODE REVIEW

File: artifactory/commands/commandssummaries/buildinfosummary.go [hunk 1]
```diff
@@ -46,6 +59,23 @@ func (bis *BuildInfoSummary) buildInfoTable(builds []*buildInfo.BuildInfo) strin
 	return tableBuilder.String()
 }
 
+func (bis *BuildInfoSummary) buildInfoModules(builds []*buildInfo.BuildInfo) string {
```
Comment (eyalbe4) [inline artifactory/commands/commandssummaries/buildinfosummary.go:62]: Let's add a unit test for this function, validating the output, including the case of stripping unsupported module types.

File: artifactory/commands/commandssummaries/buildinfosummary.go [hunk 2]
```diff
@@ -55,3 +85,27 @@ func parseBuildTime(timestamp string) string {
 	// Format the time in a more human-readable format and save it in a variable
 	return buildInfoTime.Format(timeFormat)
 }
+
+func (bis *BuildInfoSummary) generateModuleMarkdown(module buildInfo.Module) string {
```
Comment (eyalbe4) [inline artifactory/commands/commandssummaries/buildinfosummary.go:89]: Let's add a unit test for this function.

File: artifactory/utils/utils.go [hunk 3]
```diff
@@ -298,3 +300,13 @@ func GetTestDataPath() (string, error) {
 	}
 	return filepath.Join(dir, "testdata"), nil
 }
+
+func GetRtMajorVersion(servicesManager artifactory.ArtifactoryServicesManager) (int, error) {
```
Comment (eyalbe4) [inline artifactory/utils/utils.go:304]: Let's add a unit test for this function.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: artifactory/commands/commandssummaries/buildinfosummary.go (hunk 2 above)
Comment (RobiNino) [reply artifactory/commands/commandssummaries/buildinfosummary.go:89]: tested as part of `buildInfoModules`

Comment (eyalbe4) [reply artifactory/commands/commandssummaries/buildinfosummary.go:89]: OK @RobiNino.

**Pull request merged**